client/components/tip-box/style.scss

```diff
@@ -33,6 +32,14 @@
 			color: #0063a1; // Gutenberg/Blue -10%
 		}
 	}
+	&.yellow {
+		fill: $gray-50;
+		background-color: #fef8ee;
+		color: $gray-900;
+		.components-button.is-link {
+			color: #0063a1; // Gutenberg/Blue -10%
```

Comment:
Is this comment/colour code correct?

Just double checking because it's the same as the `.gray` CSS defined above 🙂 I guess they probably use the same link colour, but wanted to make sure!